.ruby-version

```diff
@@ -0,0 +1 @@
+2.5.0
```

Comment:
Should this be `2.6.0`, given the Dockerfile below?

Also, 2.6 only just came out, so I'm not sure if it has any issues (e.g. with gems).

Reply:
2.5.0 is required for it to run in Lambda, which is what the lock file was generated against. There's a v2.5.1 docker image I could use but it seemed fine with the latest image.

Reply:
Oh there is an official 2.5.0 image! I just didn't see it in the tag list. I'll lock it down properly to this version so it'll work for both deployment strategies (lambda tied to v2.5.0 for now).